src/structures/APIMessage.js

```diff
@@ -82,7 +82,7 @@ class APIMessage {
     if (this.options.content === null) {
       content = '';
     } else if (typeof this.options.content !== 'undefined') {
-      content = Util.resolveString(this.options.content);
+      content = Util.verifyString(this.options.content, new RangeError('MESSAGE_CONTENT_TYPE'), false);
```

Comment:
I assume disallowing empty strings here is to require a more explicit `null` to clear the message's content?
(When having a file or embed on the message, that is a valid action)

Reply:
that might actually just be an oversight, will allow empty strings

Reply:
Psst again